packages/aws-cdk/lib/legacy-aws-auth.ts

```diff
@@ -25,7 +25,7 @@ export function cached<A extends object, B>(obj: A, sym: symbol, fn: () => B): B
  * @deprecated
  */
 export interface ContextProviderPlugin {
-  getValue(args: {[key: string]: any}): Promise<any>;
+  getValue(args: { [key: string]: any }): Promise<any>;
```

Comment:
This wasn't picked up before because the default eslint `key-spacing` rules doesn't do TypeScript.